week1/assignment/index.css

```
@@ -0,0 +1,190 @@
+body{
+    margin: 0;
+    padding: 0;
+    background-color: black;
+}
+
+.background-image{
+    position: relative;
+    width: 100%;
+    height: 100vh;
+}
+
+.nav-topbar{
+    position: fixed;
+    display: flex;
+    z-index: 1;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.topbar-left{
+    display: flex;
+    justify-content: flex-start;
+    list-style-type: none;
+    color: white;
+    width: 50%;
+    align-items: center;
+}
+
+.topbar-right{
+    display: flex;
+    justify-content: flex-end;
+    list-style-type: none;
+    color: white;
+    width: 50%;
+}
+
+.logo{
+    font-size: 35px;
+    margin-right: 25px;
+    font-weight: bold;
+}
+
+.item a{
+    font-size: 16px;
+    color: white;
+    margin-right: 20px;
+    text-decoration: none;
+}
+
+.emoji i{
+    margin-right: 20px;
+}
+
+.semi-title{
+    padding: 0px 20px;
+    position: absolute;
+    z-index: 10;
+    background-color: rgba(255, 255, 255, 1);
+    right: 30px;
+    bottom: 100px;
+    border-radius: 10px;
+}
+.semi-title h3 {
+    color: #0d92f4;
+}
+
+.link{
+    position: absolute;
+    right: 30px;
+    bottom: 40px;
+    z-index: 10;
+    background-color: rgba(255, 255, 255, 0.3);
+    color:white;
+    padding: 10px;
+    border-radius: 10px;
+}
+
+.link:hover{
+    background-color:rgba(255, 255, 255, 0.1) ;
+    cursor: pointer;
+}
+
+.link a {
+    text-decoration: none;
+    color:white;
+}
+
+.link i {
+    size: 30px;
+}
```

p2: 이 부분 `size`가 맞을까요?? 적용이 안되는 것 같습니다!! 
제가 알기로는 `size`라는 속성은 없는 것으로 알고 있어서, `font-size`로 수정 부탁드립니다!!
```suggestion
.link i {
    font-size: 30px;
}
```

---

적용이 안 되고 있었네요 감사합니다!